### DISCUSSION TITLE
LINK : fatal error LNK1104: cannot open file 'CORE_RL_coders_.lib'

### DISCUSSION THREAD

**Discussion by Plamonster** [dlemstra/Magick.Native#16] [General]
Hi. I want to use the Magick.Net API. 
Company policies are that we have to download the source code and build it on our build farm. I'm am not allowed, still by my company policy, to generate a read:packages token for Magick.Native and so I can't successfully run the Magick.Net install.ps1 script.

So I downloaded the Magick.Native source code to build it. which also mean to download the source code of ImageMagick. 
I successfully built the ImageMagick.
But I'm not able to build Magick.Native. I got the LINK : fatal error LNK1104: cannot open file 'CORE_RL_coders_.lib' error. I didn't have any CORE_RL_coders_ library generated by the ImageMagick build. Can't find any CORE_RL_coders_ file on my HD.

Any hint?
tx!

Windows 10.
Visual Studio 2017
ImageMagick target setup :
- Dynamic (DLL) builds
- Enable HDRI
- Inlcude demo...
- Exclude deprecated methods
- Q16
- x86
- VS2017

**Comment by dlemstra** [comment]
My advise would be use the steps that are done in the GitHub actions build and not build ImageMagick yourself in a separate solution. And you will probably also need to use a more recent version of Visual Studio to get support for all the features that are now available in Magick.NET.

**Reply by Plamonster** [reply]
Ok, tx for the quick answer, I'll give it a try. I can have VS 2019 on my desktop, but not on our build farm :(

**Reply by Plamonster** [reply]
I was able to build Magick.Native following the GitHub actions, thanks for the hint. I did the same for Magick.Net but unfortunately, I wasn't able to succeed with VS2017 as the build requires some .Net version not available in 2017. I installed VS 2019 and it successfully built Magick.Net.
I will have to talk with my boss for VS2019!
Merci!
